en/docs/learn/configuring-saml2-web-single-sign-on.md

```diff
@@ -123,6 +123,17 @@ To configure SAML2 Web SSO:
     <tr class="even">
     <td><strong>Certificate Alias</strong></td>
     <td><div class="content-wrapper">
+    <p class="admonition-title">Tip</p>
+    <p>From WSO2 IS 5.12.0 onwards, the <strong>Certificate Alias</strong> option would be hidden from the application
+     configurations. To enable the <strong>Certificate Alias</strong> option in the console UI, add the following
+      configuration to the <code>                &lt;IS_HOME&gt;/repository/deployment.toml</code>.
+      
+      ```toml
```

Comment:
Please check that the alignment is as follows: 
```toml
[console.applications.ui]
certificate_alias_enabled=true
```

Reply:
Addressed with https://github.com/wso2/docs-is/pull/2403/commits/5fead90dcb7a5569141bd5cff0b204d774fa79e6.